package/fedora/freecad.spec

```
@@ -111,16 +126,25 @@ BuildRequires:  libappstream-glib
 # here.
 Requires:       %{name}-data = %{epoch}:%{version}-%{release}
 # Obsolete old doc package since it's required for functionality.
-Obsoletes:      %{name}-doc < 0.13-5
+#%if %{version} > 0.22
+Obsoletes:      %{name}-doc < 0.22-1
+#%else
+#Obsoletes:      %{name}-doc < 0.13-5
+#%endif
 Requires:       hicolor-icon-theme
 
 Requires:       fmt
 
 Requires:       python3-pivy
 Requires:       python3-matplotlib
 Requires:       python3-collada
-Requires:       python3-pyside2
+%if 0%{?fedora} > 38
+Requires:       python3-pyside6
+Requires:       qt6-assistant
+%else
+Requires:       python3-pyside5
```

```suggestion
Requires:       python3-pyside2
```
although like said above I would just remove support for fedora < 40

---

Will fix it.

---

Only non-EOL versions supported, so versions removed.